fs/dafsserver/dafsserver.cpp

```diff
@@ -160,12 +160,26 @@ static ISecureSocket *createSecureSocket(ISocket *sock, bool disableClientCertVe
     return secureContextServer->createSecureSocket(sock, loglevel);
 }
 #else
-static ISecureSocket *createSecureSocket(ISocket *sock)
+static ISecureSocket *createSecureSocket(ISocket *sock, bool disableClientCertVerification)
 {
     throwUnexpected();
 }
 #endif
 
+static void reportFailedSecureAccepts(const char *context, IException *exception, unsigned &numFailedConn, unsigned &timeLastLog)
+{
+    numFailedConn++;
```

Comment:
@mckellyln I think it's fine, but would note that for a given log entry, there is a slight chance that we get a bunch of "row service" context errors and then after the time threshold report on a "secure" context error.  One may be caused by load-balancer port checking and the other by the issues Jake had mentioned. But if there are mulitple of these in the logs that should wash out.